Homework 03/AZ/3.Aufgabe_27.01.22.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+# Create two time series: activity data (AD) and emission factor (EF)
+# Select start and stop years (e.g. 1990 to 2020)
+# Fill time series with random values for all years
+  # AD values should be 100 +- random 20
+  # EF values should be 0.5 +- random .3
+# Calculate (EMyear = ADyear * EFyear) and store emissions in new time series (EM)
+# Print yearly and total results
+
+
+# In[ ]:
+
+
+# steps: 
+# create AD list with values between 20 and 100 (with a step size of 1)
+# create EF list with values between 0.5 and 3 ( with a step size of 1)/other step size?
+# create year list from 1990 - 2021 with a step size of 1 year
+# multiplication of  EM = EF * AR
+# merge  of EM and year in a dictionary
+
+
+# In[1]:
+
+
+import random
+import numpy as np
+
+
+# In[2]:
+
+
+# generation of 3 lists with start and end value and step size
+AR = [random.randrange(20, 100, 1) for i in range(32)]
+EF = [np.random.uniform(0.5, 3, 1) for i in range(32)]
+year = list(range(1990, 2022, 1))
```

Comment:
`step` of one is implied and could be omitted (see above as well)

```suggestion
year = list(range(1990, 2022))



```